.travis.yml

```diff
@@ -1,22 +1,13 @@
 sudo: false
 language: node_js
 node_js:
-- '0.12'
+- '5.9'
```

Comment:
I didn't realize v5 was a thing. Looks like the right release line for Billimanjaro.